public/app/features/plugins/sandbox/sandbox_plugin_loader.ts

```diff
@@ -213,7 +213,21 @@ async function doImportPluginModuleInSandbox(meta: SandboxPluginMeta): Promise<S
   });
 }
 
-function resolvePluginDependencies(deps: string[], pluginId: string) {
+/**
+ *
+ * This function resolves the dependencies using the array of AMD deps.
+ * Additionally it supports the RequireJS magic modules `module` and `exports`.
+ * https://github.com/requirejs/requirejs/wiki/Differences-between-the-simplified-CommonJS-wrapper-and-standard-AMD-define#magic
+ *
+ */
+function resolvePluginDependencies(deps: string[], pluginMeta: SandboxPluginMeta) {
+  const exports = {};
+  const module: ModuleMeta = {
```

Comment:
I recommend you to change this variable name to something else to avoid confusions with the variable `module` that already exists inside this context from webpack.

Maybe something like `const pluginModuleDep: ModuleMeta = {` and `const pluginExports = {}`

Reply:
Updated var names in 411f6ac28bfe9b486e4054c4d5908a722aecb22b. 👍